### DISCUSSION TITLE
BlueSCSI images disappearing

### DISCUSSION THREAD

**Discussion by real-AP** [erichelgeson/BlueSCSI-v1#269] [Q&A]
Hi.  I recently got a bluescsi v2 desktop with Wi-Fi.  I also got a ZuluSCSI.  I intended to use one of them for a PowerCenter Pro 210, using the motherboard SCSI (because BeOS doesn't support bundled SCSI card).

I have successfully used images created with  OS X terminal, dickjockey, and pre made blank images from the links provided.  I have formatted them with Apple HD SC, and FWB toolkit.  They worked for me for booting BeOS, MacOS 7.6 and 8.6.

However, sometimes when I change startup disk and reboot, or just reboot, or shut down and start up. I get the ? Floppy on startup.  I go to check out the SD card (Sandisk extreme pro 128GB), and there are images that have just disappeared.  Not hidden or in the trash, just gone.  Sometimes the SD card has even become unreadable by OS X, and I need to reformat.  I also tried fully formatting with the SD card format utility.

I have done all of these things with the ZuluSCSI V1.2, and have not had the same issue.  Same images, and I can do any combination of rebooting or formatting of drives without issue.

Any ideas?

**Comment by erichelgeson** [comment]
We use a library called SDFat as SD cards - and have seen many odd issues with SanDisk - it's our only SD brand on our [Avoid](https://github.com/BlueSCSI/BlueSCSI-v2/wiki/SD-Cards#avoid) list. First step is always to do a full overwrite format with the SD Card Formatting tool - this many times is all you need to do to re-align everything on the SD card's flash memory. If that doesn't resolve your issue please continue with the [Troubleshooting guide](https://github.com/BlueSCSI/BlueSCSI-v2/wiki/Troubleshooting#what-to-try-first) and report back after you've fully gone through it.

**Reply by real-AP** [reply]
Oh.  I see that now.  I have done many of the steps on the troubleshooting guide where possible, and didn't see anything about my issue here or in the troubleshooting section.  I did just now see a Reddit post about someone having a similar issue.

the one thing I didn't try is a non Sandisk SD card.  I tried a 8GB industrial Sandisk with same issues.  I will try out a different brand soon and see if that solves my issues.

Thanks, I looked at that section but didn't notice the avoid Sandisk statement.
